packages/api/public/spec/paths/language.yml

```diff
@@ -12,7 +12,7 @@ get:
     "200":
       description: Ok
       content:
-        application/vnd.api+json:
+        application/json:
```

Comment:
I had to revert to `application/json` which isn't what JSON:API expects, but nextjs didn't have a good way to handle a custom content-type